infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -113,9 +113,17 @@ export class AtatWebApiStack extends cdk.Stack {
     portfolio.addMethod("POST", new apigw.LambdaIntegration(createPortfolioStepFn));
     table.grantReadWriteData(createPortfolioStepFn);
 
+    // getPortfolioStep
+    const getPortfolioStepFn = new lambdaNodejs.NodejsFunction(this, "GetPortfolioStepFunction", {
+      entry: "applications/portfolioDrafts/portfolio/getPortfolioStep.ts",
+      ...sharedFunctionProps,
+    });
+    portfolio.addMethod("GET", new apigw.LambdaIntegration(getPortfolioStepFn));
+    table.grantReadData(getPortfolioStepFn);
+
     // TODO: getPortfolioDrafts
-    // TODO: getPortfolioDraft
     // TODO: getPortfolioStep
+    // TODO: getPortfolioDraft
```

Comment:
I think you meant to delete getPortfolioStep?

Reply:
Yes